### PR TITLE
Add HandlerReturn type

### PR DESCRIPTION
**Pull request by zyvz**

### CODE REVIEW

**Review by nweber-gh — APPROVED** [review]

File: index.d.ts [hunk 1]
```diff
@@ -266,3 +266,5 @@ export function isLoop(test: any): boolean;
 export function getModel<S>(loop: S | Loop<S>): S;
 
 export function getCmd(a: any): CmdType | null;
+
+export type HandlerReturn<S> = S | Loop<S>;
```
Comment (bdwain) [inline index.d.ts:270]: should we call it ReducerHandlerReturn? Might make it more clear

Comment (zyvz) [reply index.d.ts:270]: Its kinda wordy but I'm impartial

Comment (nweber-gh) [reply index.d.ts:270]: The most idiomatic is probably `LoopReducerHandlerFnReturn`, though I agree with Baruch that we're getting 'wordy. `HandlerFnReturn`? `HandlerFunctionReturn`? I dunno. I'm mostly impartial. Maybe a comment above the type would help?

Comment (bdwain) [reply index.d.ts:270]: Well i thought the name change was more helpful for the user of the library, not the author. Maybe let's leave it and we can change it if there's feedback

**Pull request merged**